postgres/codenamemap.yaml

```diff
@@ -46,7 +48,7 @@
 
 {{ codename|default(name, true) }}:
   # PostgreSQL packages are mostly downloaded from `main` repo component
-  fromrepo: {{ name }}
+  fromrepo: {{ fromrepo }}
```

Comment:
This really needs to be fixed. The `fromrepo` for "downstream" installations should be `None` by default, but for the upstream it should match the `pkg_repo.name` value across RedHat-based distros. I see that it is already done at upper mapping level in the OS family map. So we can just remove manipulations on `fromrepo` here.

Reply:
Yes, its handled correctly for "upstream" repos in OS family map file.  The `fromrepo: ''` is defaults.yaml should evaluate to None,  for Debian it should be `name` but for Fedora/Redhat what should the "downstream" fromrepo value be?

Reply:
I think we should we that default and does not handle the `fromrepo` parameter for downstream at all. That's rarely applicable usecase and would be to complex to handle for now.

Reply:
Okay, so `fromrepo` is not used for Fedora. For Ubuntu I do need to set `fromrepo` jinja variable to some value, either `{% set fromrepo = name %}` or `{% set fromrepo = '' %}` or `{% set fromrepo = None %}` in codenamemap.yaml.  The SLS files will check `{% if postgres.fromrepo }` so just need to set correct value so condition fails.  Is this how you see it too?